galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceci.manager.ivt/src/main/java/dev/galasa/cicsts/ceci/manager/ivt/CECIManagerIVT.java

```diff
@@ -341,6 +430,46 @@ public void testLinkToProgramToCommarea() throws CeciException, CicstsManagerExc
       assertThat(outputData).isUpperCase();
       assertThat(outputData).startsWith(variableValue.toUpperCase());
    } 
+   
+   /**
+    * Tests that the EIB fields are retrieved correctly.
+    * 
+    * @throws CicstsManagerException 
+    * @throws CeciException
+    * @throws FieldNotFoundException 
+    * @throws NetworkException 
+    * @throws TerminalInterruptedException 
+    * @throws KeyboardLockedException 
+    * @throws TimeoutException 
+    */
+   @Test
+   public void testGetEIBFields() throws CeciException, CicstsManagerException, TimeoutException, KeyboardLockedException, TerminalInterruptedException, NetworkException, FieldNotFoundException {
+	   logger.info("Testing that important fields from the Exec Interface Block are retrieved correctly");
+	   
+	   cics.ceci().issueCommand(ceciTerminal, "PUT CONTAINER(BOB) CHANNEL(BOB) FROM('HELLO')");
+	   IExecInterfaceBlock eib = cics.ceci().getEIB(ceciTerminal);
+	  
+	   assertThat(eib.getResponse()).isEqualTo("NORMAL");
+	   assertThat(eib.getEIBRESP()).isEqualTo(0);
+	   assertThat(eib.getEIBRESP2()).isEqualTo(0);
```

Comment:
As these are likely to be default at zero it would be good to test them with a non-zero value - this could be done by linking to a program that doesn't exist.